packages/contractkit/src/wrappers/VestingInstanceWrapper.ts

```diff
@@ -0,0 +1,305 @@
+import { Signature } from '@celo/utils/lib/signatureUtils'
```

Comment:
Looks like we're missing the wrappers for various voting actions. Fine for these to be included in a follow-up PR, our focus right now should be the contracts and unit tests.

Reply:
so was the plan as we last spoke